menhir-recover/synthesis.ml

```diff
@@ -160,8 +160,9 @@ struct
       constant is the same for all branches, $\kappa_{i,j} = \kappa_i$,
   *)
 
-  let cost_of_prod p = Cost.add (Cost.of_int 1) (A.cost_of_prod p)
-  let cost_of_symbol s = Cost.add (Cost.of_int 1) (A.cost_of_symbol s)
+  (* TODO: update doc *)
```

Comment:
TODO

Reply:
fixed. I've just removed it because I left a comment in the attribtues and cannot find any other outdated docstrings.